app/services/scrapers/firecrawl_client/client.py

```
@@ -0,0 +1,95 @@
+    @classmethod
+    def reset_instance(cls) -> None:
+        """测试用：重置单例。"""
+        with cls._lock:
+            cls._instance = None
+
+    def scrape_url(
+            self,
+            url: str,
+            formats: Optional[List[str]] = None,
+            only_main_content: bool = True,
+            timeout_seconds: Optional[int] = None,
```

Ruff (0.14.13)

Unused method argument: `timeout_seconds`

(ARG002)